### DISCUSSION TITLE
Unable to upload photo/choose proof type/switch camera in application window.

### DISCUSSION THREAD

**Discussion by ZYWWYZ123** [community/community#138957] [GitHub Education]
### Hi everyone,

Hi all, 
Hello GitHub Education support team,
I have some difficulties while applying for Students Developer Pack, especially when it requires me to upload my proof:
- There is no “upload” button in where it should be. The only I saw is the camera button. I tried using virtual camera to capture and upload image (including all student status information required ), but it seems the system can’t recognize the characters correctly and I was rejected at once.
- Also according to the guidance document, there should be a button to choose proof type, but I saw nothing there.
- When I tried to switch camera, it doesn’t work properly and nothing appears. Clicking the “take photo “ button only results in a blank image. I tired on my iPhone and iPad, unfortunately, same thing happened again and again. Only camera in front of my device (also, that’s the default camera the webpage uses ) correctly working, which makes it very difficult to take a clear photo.

I tried on my laptop, iPhone and iPad, but problems above always happen and they confused me a lot. 
Thank you for your help and guidance! I’m looking forward to solve these problems and complete my application successfully.

Best wishes.

**Comment by samueltexa** [comment]
Hi @ZYWWYZ123,
Clear your browser’s cache and cookies, as old or corrupted files might be causing the upload and camera issues.

**Comment by chungyu1108** [comment]
try using an updated browser, clearing cache, and ensuring camera permissions are granted. If the camera isn't working, switching devices or using a different browser may help. You can also contact GitHub support directly with detailed information and screenshots to resolve the issue. Hopefully, this will allow you to complete your application smoothly.

**Reply by KamcyAhaka** [reply]
Hi @chungyu1108, do you happen to know a way to contact GitHub Support on the free plan? I am experiencing the same issues as @ZYWWYZ123 and I've done everything you suggested to no success - updated browser, multiple browsers, different devices, cleared app data.

**Comment by KamcyAhaka** [comment]
Hi @ZYWWYZ123 , were you able to fix this issue? If so how did you fix it?

**Reply by ZYWWYZ123** [reply]
Hi @KamcyAhaka. Sorry but I really don’t know how to fix it😢. I just printed the document out and eventually used the front camera to take a photo.

**Reply by KamcyAhaka** [reply]
That sucks man. Out of curiosity, how did you take a photo with the front camera and how was the output? I imagine it would have taken some gymnastics to get the right angle and focus.

**Reply by ZYWWYZ123** [reply]
Just tired many times😢. It really took me some time to get a good photo... Hope they can notice and fix this bug😵‍💫

**Reply by ahmadrazacdx** [reply]
> That sucks man. Out of curiosity, how did you take a photo with the front camera and how was the output? I imagine it would have taken some gymnastics to get the right angle and focus.

I've applied to student pack facing same issue and due to some quality and orientation issues with front camera, my application got declined as well 🤧

**Reply by KamcyAhaka** [reply]
> Just tired many times😢. It really took me some time to get a good photo... Hope they can notice and fix this bug😵‍💫

I sent them a message via LinkedIn, if you can, I'd recommend doing so too. If many people bring it to their attention, it might push them to fix the bug.

**Comment by ahmadrazacdx** [comment]
I'm also facing the same issue 😭

**Reply by KamcyAhaka** [reply]
The only way I found to communicate with them was via LinkedIn. If you can, send them a message there as well, if enough people can bring it to their attention, it might push them to fix the bug.
